SyntheticsRunTestsTask/__tests__/main.test.ts

```diff
@@ -33,15 +33,15 @@ describe('Test suite', () => {
     })
 
     expect(task.succeeded).toBe(true)
-    expect(task.warningIssues.length).toEqual(0)
+    // expect(task.warningIssues.length).toEqual(0)
```

Comment:
Why not expecting the warnings?
```suggestion
    expect(task.warningIssues).toEqual([...])
```